doc/cookbook/engine.rst

```diff
@@ -0,0 +1,36 @@
+Engines
+=======
+
+Ploomber currently supports 2 engines for running notebooks:
+
+- Ploomber Engine
+- Papermill Engine
+
+
+`Ploomber Engine <https://engine.ploomber.io/en/latest/quick-start.html>`_
```

Comment:
please add sample YAMLs of how choosing one or another would look like.

Also show how to pass parameters to them.